Core/Core/Credentials/Account.cs

```diff
@@ -22,7 +25,7 @@ public string id
       {
         if (serverInfo == null || userInfo == null)
         {
-          throw new SpeckleException("Incomplete account info: cannot generate id.");
+          throw new InvalidOperationException("Incomplete account info: cannot generate id.");
```

Comment:
Just wondering if we had any usages here were we try/catched SpeckleException, we should update those.

Reply:
After a quick check, I couldn't find anywhere we are.